### ISSUE TITLE
AppleGVA (Hardware decoding/encoding) works on Navi 22 but not DRM

### ISSUE THREAD

**Issue by VisualEhrmanntraut** [ChefKissInc/NootRX#4]
**Describe the bug**
AppleGVA (Hardware decoding/encoding) using VCN works on Navi 22 but not DRM; you usually just get audio but no video.

**To Reproduce**
1. Try DRM playback
4. See error or video but no audio

**Expected behaviour**
The DRM video to play normally.

**Comment by ExtremeXT** [comment]
Fairplay 2.x/3.x based hardware DRM content seems to be playing just fine, only Fairplay 1.x (QuickTime DRM), Fairplay 4.x (a mix of all previous Fairplay versions) (Apple TV, Apple Music Bluetooth / Loseless) seems to be broken. Might be related to HDCP.

**Comment by Doghouse-Mike** [comment]
Not sure if it helps, but I noticed Netflix (via Firefox) would do the black screen *with* audio when I also had another device connected to the "Mac" via vnc. Displays video fine when no VNC shenanigans are going on.

**Comment by VisualEhrmanntraut** [comment]
Yes, it's similar to NootedRed's behaviour where the audio plays but the video is black.

**Comment by ExtremeXT** [comment]
@Doghouse-Mike Firefox uses Software DRM, so this is unrelated to NootRX. Not playing DRM content when a remote connection is in progress is likely an intended feature to prevent recording.

**Comment by maksimlenskii** [comment]
Am I correct in understanding that I was moved to this thread to follow progress regarding DRM issues? I believe that WhateverGreen somehow helped to bypass this problem with other video cards?

**Comment by maksimlenskii** [comment]
I forgot to write that the video card test in Cinebench 2024 also does not start

**Comment by maksimlenskii** [comment]
Maybe this will help. but I changed the system from MacPro 7.1 to imac 20.2 and I was able to play ALAC (and any other) music on Apple Music; Apple TV is still just announcements
RX6750GRE

**Comment by VisualEhrmanntraut** [comment]
@maksimlenskii a missing VideoToolbox patch was added last week, does the latest commit have the playback working on MacPro7,1?

**Comment by maksimlenskii** [comment]
> @maksimlenskii a missing VideoToolbox patch was added last week, does the latest commit have the playback working on MacPro7,1?
Apple TV unfortunately does not work yet

**Comment by KiddyV** [comment]
> @maksimlenskii a missing VideoToolbox patch was added last week, does the latest commit have the playback working on MacPro7,1?

The new version seems to have improved, but when using FCPX to export videos after the computer wakes up from sleep, the video interface cannot be displayed through Safari video web pages, and the video part is displayed in red; Google Chrome does not have this situation. This situation only occurs when the computer has undergone sleep.

**Comment by actuallyaridan** [comment]
> @maksimlenskii a missing VideoToolbox patch was added last week, does the latest commit have the playback working on MacPro7,1?

I don't know if this is related but Netflix on Safari still returns error code S7706 (No DRM).

**Comment by maksimlenskii** [comment]
I gave up)) I moved to RX6800XT
